util/wireshark/minetest.lua

```diff
@@ -1299,7 +1300,6 @@ do
 		t:set_text("Minetest, Peer: " .. buffer(4,2):uint() .. ", Channel: " .. buffer(6,1):uint())
 
 		local reliability_info
-		local pos
```

Comment:
should not be removed

Reply:
Had to step away. Oops, a blunder, fixed.